### PR TITLE
ring localisations and rational numbers

### PR DESCRIPTION
**Pull request by plt-amy**
numbers :-1:

### CODE REVIEW

**Comment by Lavenza** [comment]
[Pull request preview](https://preview.1lab.dev/429)

File: src/Algebra/Ring/Localisation.lagda.md [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+In the literature, this is more commonly phrased as $u(xt - ys) = 0$,
+but the equivalence between that and our definition is routine.
+
+```agda
+  _≈'_ : ∀ {ℓ'} {S : ⌞ R ⌟ → Type ℓ'} → Fraction S → Fraction S → Type _
+  _≈'_ {S = S} (x / s) (y / t) = ∃[ u ∈ R ] (u ∈ S × u * (x * t - y * s) ≡ 0r)
+```
+
+<details>
+<summary>The calculation can be found in this `<details>`{.html} block.</summary>
```
Comment (ncfavier) [inline src/Algebra/Ring/Localisation.lagda.md:166]: Suggestion:

````suggestion
<details>
<summary>
In the literature, this is more commonly phrased as $u(xt - ys) = 0$,
but the equivalence between that and our definition is routine.

```agda
  _≈'_ : ∀ {ℓ'} {S : ⌞ R ⌟ → Type ℓ'} → Fraction S → Fraction S → Type _
  _≈'_ {S = S} (x / s) (y / t) = ∃[ u ∈ R ] (u ∈ S × u * (x * t - y * s) ≡ 0r)
```
</summary>
````

**Review by ncfavier — APPROVED** [review]

File: src/Algebra/Ring/Localisation.lagda.md (hunk 1 above)
Comment (plt-amy) [reply src/Algebra/Ring/Localisation.lagda.md:166]: inside of a <summary> is phrasing content so this is very illegal https://html.spec.whatwg.org/#the-summary-element

Comment (ncfavier) [reply src/Algebra/Ring/Localisation.lagda.md:166]: Oops. Didn't know that.

**Review by ncfavier — APPROVED** [review]

File: src/Algebra/Ring/Localisation.lagda.md (hunk 1 above)
Comment (plt-amy) [reply src/Algebra/Ring/Localisation.lagda.md:166]: yeah i had tried *exactly this* and found out :stuck_out_tongue:

**Pull request merged**